conf/viewvc.conf.dist

```diff
@@ -336,6 +336,19 @@
 ##
 #diff = 
 
+## max_context: Number of context lines to pass to diff program with
+## "--unified" option, for "Full human readable" format.
+##
+## The GNU diff utility does not have option to get whole context
+## or unified diffs without omission.  Alternatively, we indicate
+## large number of context lines practically enough.
+##
+## The default value is 10000, which is large enough for diff text
+## on web browser, we think. You can increase or decrease this value
+## as long as the diff utility can accept it.
+##
```

Comment:
I apologize -- I'm trying not to be overly critical of the English grammar, but as this text is more "visible", I'd like it as consistent as possible.  What's mostly lacking are the definite and indefinite articles.

```
## max_context: Number of context lines passed to GNU diff's --unified
## option when displaying ViewVC's "Full human readable" diff format.
##
## The GNU diff utility does not have the option to request all the context
## of unified diffs, so the best we can do is to ask for a very large
## number of context lines.  The default value is 10000, which should be
## large enough to result in the display of all context lines for any files
## whose diffs you'd care to even see in a web browser.  Of course, you can
## increase or decrease this value as best suits your server.

Reply:
Also, done in [6f7c0d2](https://github.com/viewvc/viewvc/pull/297/commits/6f7c0d2f1db140e49a50ef945eaca188f16bfbde)